daemon/daemon.go

```diff
@@ -973,6 +974,20 @@ func (daemon *Daemon) setHostConfig(container *Container, hostConfig *runconfig.
 
 	container.Lock()
 	defer container.Unlock()
+
+	var stat syscall.Stat_t
```

Comment:
FWIW - this is identical to how Windows handles apparmor (lack of) through the call to parseSecurityOpt